kubemarine/patches/disable_token_automount.py

```diff
@@ -0,0 +1,34 @@
+from textwrap import dedent
+from io import StringIO
+
+from kubemarine.core.action import Action
+from kubemarine.core.patch import RegularPatch
+from kubemarine.core.resources import DynamicResources
+from kubemarine import kubernetes, plugins
+from kubemarine.plugins.manifest import Processor, EnrichmentFunction, Manifest, Identity
+ 
+class TheAction(Action):
+    def __init__(self) -> None:
+        super().__init__("Disable automount of Serice acocunt tokens")
+    
+    def run(self, res: DynamicResources) -> None:
+        cluster = res.cluster() 
+        cluster.log.debug("As part of this patch, all the plugins will be redeployed")
+        plugins.install(cluster)      
```

Comment:
I think we should install only affected OOB plugins, not including the custom ones.

Reply:
logic modified to redeploy only OOB plugins through the provided patch.